bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -83,72 +186,125 @@ def post_block(self, block: Block) -> HexStr:
 
+    def post_block_raw(self, block: Block) -> HexStr:
+        """Returns the BlockId of the submitted block.
+        POST /api/core/v3/blocks
+
+        Returns:
+            The corresponding block id of the block.
+        """
+        return self._call_method('postBlockRaw', {
+            'block': block
+        })
+
     def get_block(self, block_id: HexStr) -> Block:
-        """Get the block corresponding to the given block id.
+        """Finds a block by its ID and returns it as object.
+        GET /api/core/v3/blocks/{blockId}
+
+        Returns:
+            The corresponding block.
         """
         return Block.from_dict(self._call_method('getBlock', {
             'blockId': block_id
         }))
 
+    def get_block_raw(self, block_id: HexStr) -> List[int]:
+        """Finds a block by its ID and returns it as raw bytes.
+        GET /api/core/v3/blocks/{blockId}
+
+        Returns:
+            The corresponding raw bytes of a block.
+        """
+        return self._call_method('getBlockRaw', {
+            'blockId': block_id
+        })
+
     def get_block_metadata(self, block_id: HexStr) -> BlockMetadata:
-        """Get the block metadata corresponding to the given block id.
+        """Returns the metadata of a block.
+        GET /api/core/v3/blocks/{blockId}/metadata
+
+        Returns:
+            The corresponding block metadata.
         """
         return BlockMetadata.from_dict(self._call_method('getBlockMetadata', {
             'blockId': block_id
         }))
 
     def get_block_with_metadata(self, block_id: HexStr) -> BlockWithMetadata:
-        """Get a block with its metadata corresponding to the given block id.
+        """Returns a block with its metadata.
+        GET /api/core/v2/blocks/{blockId}/full
+
+        Returns:
+            The corresponding block with it metadata.
         """
         return BlockWithMetadata.from_dict(self._call_method('getBlockWithMetadata', {
             'blockId': block_id
         }))
 
-    def get_block_raw(self, block_id: HexStr) -> List[int]:
-        """Get the raw bytes of the block corresponding to the given block id.
-        """
-        return self._call_method('getBlockRaw', {
-            'blockId': block_id
-        })
+    # UTXO routes.
 
-    def post_block_raw(self, block_bytes: List[int]) -> HexStr:
-        """Post a block as raw bytes.
+    def get_output(
+            self, output_id: Union[OutputId, HexStr]) -> OutputWrapper:
+        """Finds an output by its ID and returns it as object.
+        GET /api/core/v3/outputs/{outputId}
 
         Returns:
-            The corresponding block id of the block.
+            The corresponding output.
         """
-        return self._call_method('postBlockRaw', {
-            'blockBytes': block_bytes
-        })
+        output_id_str = output_id.output_id if isinstance(
+            output_id, OutputId) else output_id
+        return OutputWrapper.from_dict(self._call_method('getOutput', {
```

Comment:
getOutput returns `OutputWithMetadataResponse` in the bindings core which has only the output and metadata and not the outputid proof :thinking:

Reply:
Fixed